content/blog/a-brief-tour-of-kusama-asset-hub.md

```diff
@@ -0,0 +1,39 @@
+---
+date: 2024-02-15
+tags: Polkadot Asset Hub Statemint
+image: '/blog/a-brief-tour-of-kusama-asset-hub.webp'
+title: 'A brief tour of Asset Hub'
+subtitle: 'The chain that puts the community first.'
+---
+
+## What is the Polkadot Asset Hub?
+
+Asset Hub (formerly Statemint) is a parachain on Polkadot for creating and sending fungible and non-fungible tokens, with the native token being DOT. Its Kusama equivalent is the similarly named Kusama Asset Hub (formerly Statemine). It connects directly to the Relay Chain, enabling easy exchange of information and assets between parachains. This sounds nice and all, but how is it any different from other chains on the Polkadot network?
+
+## Why Asset Hub?
+
+Asset Hub is a "common good" parachain, meaning it exists primarily for the benefit of the community in various ways.
+- The chain does not use smart contracts. Instead, all the execution details and expenses of an operation on an asset are encoded on the chain already. This is cheaper and more predictable in terms of fees compared to smart contracts. The fee  is determined through prior benchmarks of similar processes.
+- By being the most convenient option to deploy assets, the chain provides a large community for a busy marketplace to explore or buy from.
+- It is more efficient than the Relay chain for DOT balance transfers.
+- Low existential deposit required (minimum amount an account must have to exist on chain) compared to other chains.
+
+## Assets
+The process of creating an asset on Asset Hub is well documented [here](https://wiki.polkadot.network/docs/learn-guides-assets-create) in case you need the latest data or a more detailed reference. I will not go into too much detail.
```

Comment:
```suggestion
The process of creating an asset on Polkadot Asset Hub is well documented [here](https://wiki.polkadot.network/docs/learn-guides-assets-create) in case you need the latest data or a more detailed reference. I will not go into too much detail.
```